framework/web/AssetConverter.php

```diff
@@ -33,11 +35,15 @@ class AssetConverter extends Component implements AssetConverterInterface
      *     'styl' => ['css', '@app/node_modules/bin/stylus < {from} > {to}'],
      * ]
      * ```
+     *
+     * Note: `Yii::getAlias()` can replace alias at the start of the command only.
+     *
+     * @see https://sass-lang.com/documentation/cli/ SASS/SCSS
      */
     public $commands = [
         'less' => ['css', 'lessc {from} {to} --no-color --source-map'],
-        'scss' => ['css', 'sass {from} {to} --sourcemap'],
-        'sass' => ['css', 'sass {from} {to} --sourcemap'],
+        'scss' => ['css', 'sass --style=compressed {from} {to}'],
+        'sass' => ['css', 'sass --style=compressed --sourcemap=auto {from} {to}'],
```

Comment:
There's no `--sourcemap=auto`:

```
❯ sass --version
1.49.0 compiled with dart2js 2.15.1

❯ sass --help
Compile Sass to CSS.

Usage: sass <input.scss> [output.css]
       sass <input.scss>:<output.css> <input/>:<output/> <dir/>

=== Input and Output ===================
    --[no-]stdin               Read the stylesheet from stdin.
    --[no-]indented            Use the indented syntax for input from stdin.
-I, --load-path=<PATH>         A path to use when resolving imports.
                               May be passed multiple times.
-s, --style=<NAME>             Output style.
                               [expanded (default), compressed]
    --[no-]charset             Emit a @charset or BOM for CSS with non-ASCII characters.
                               (defaults to on)
    --[no-]error-css           When an error occurs, emit a stylesheet describing it.
                               Defaults to true when compiling to a file.
    --update                   Only compile out-of-date stylesheets.

=== Source Maps ========================
    --[no-]source-map          Whether to generate source maps.
                               (defaults to on)
    --source-map-urls          How to link from source maps to source files.
                               [relative (default), absolute]
    --[no-]embed-sources       Embed source file contents in source maps.
    --[no-]embed-source-map    Embed source map contents in CSS.

=== Other ==============================
-w, --watch                    Watch stylesheets and recompile when they change.
    --[no-]poll                Manually check for changes rather than using a native watcher.
                               Only valid with --watch.
    --[no-]stop-on-error       Don't compile more files once an error is encountered.
-i, --interactive              Run an interactive SassScript shell.
-c, --[no-]color               Whether to use terminal colors for messages.
    --[no-]unicode             Whether to use Unicode characters for messages.
-q, --[no-]quiet               Don't print warnings.
    --[no-]quiet-deps          Don't print compiler warnings from dependencies.
                               Stylesheets imported through load paths count as dependencies.
    --[no-]verbose             Print all deprecation warnings even when they're repetitive.
    --[no-]trace               Print full Dart stack traces for exceptions.
-h, --help                     Print this usage information.
    --version                  Print the version of Dart Sass.
```

Reply:
scss - dart sass, sass - ruby sass/legacy

Reply:
They dont uses together

Reply:
This is dart version that is recommended and is installed via npm.

Reply:
@samdark Поработаю Википедией: SASS это в первую очередь стандарт, как CSS или HTML. Dart Sass практически полностью поддерживает/реализует последнюю версию SASS 3. Ruby Sass поддерживает/реализует версию SASS 1.*, поэтому он практически не используется. Чтобы подчеркнуть переход на SASS 3 расширение файлов было изменено c `*.sass` на `*.scss`.

А теперь вернемся с моему PR команда для файлов `*.sass` использует Ruby Sass, а для файлов `*.sсss` использует Dart Sass. Для поддержки устаревшего кода я оставил команду для `*.sass` файлов в формате Ruby Sass.

Reply:
```
❯ sass --style=compressed --sourcemap=auto test.sass test.css
Could not find an option named "sourcemap".
```

I think this option should be removed. This way it will work with both Ruby Sass and Dart Sass.

Reply:
@samdark ты слушаешь, но не слышишь - уже давно никто не использует `*.sass` файлы, если кто-то компилит `*.sass` файлы, то в 99% случаев это устаревший проект использующий Ruby/Node Sass(`package.json`).